qml/ConnectionMenu.qml

```diff
@@ -183,6 +183,8 @@ Item {
                     text: "192.168.2.2"
                     Layout.columnSpan:  2
                     Layout.fillWidth: true
+                    // This regex matches IPV4 Addresses. From https://stackoverflow.com/a/50524903
+                    validator: RegExpValidator { regExp: /^(?!\.)((^|\.)([1-9]?\d|1\d\d|2(5[0-5]|[0-4]\d))){4}$/gm }
```

Comment:
"localhost" should be allowed.

Reply:
Good point. any other domain should be valid too (raspberrypi.local etc)

Reply:
updated the regex so it handles both options nicely.